docs/Users-Guide.md

```diff
@@ -39,22 +102,24 @@ To play with timesketch without any installation visit [demo.timesketch.org](htt
 tsctl is a command line tool to control timesketch.
```

Comment:
I look at `tsctl` as an **admin** tool, not a user tool... so maybe this should not be part of the user guide, tsctl is to do cleanup operations on the database, add users, etc, and can be used for some other functionality, but it is run on the container that is not accessible to anyone except the sysadmin, so I think this should be left out of the regular user guide...

Reply:
ok, I am fine doing that, guess this will then change this PR into a UserGuide refactor, but let me do that as it makes sense